Task2/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        { 
+            Console.WriteLine("Введите число 1.");
+            double digit1 = 0;
```

Comment:
Неудачное имя для локальной переменной. Digit по-английски значит "цифра", а не число. Лучше
```suggestion
            double number1 = 0;
```